modules/ztp-monitoring-policy-deployment-progress.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-configuring-managed-clusters-policies.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-monitoring-policy-deployment-progress_{context}"]
+= Monitoring managed cluster policy deployment progress
+
+The ArgoCD pipeline uses `PolicyGenTemplate` CRs in Git to generate the {rh-rhacm} policies and then sync them to the hub cluster. You can monitor the progress of the managed cluster policy synchronization after the assisted service installs {product-title} on the managed cluster.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. The {cgu-operator-first} applies the configuration policies that are bound to the cluster.
++
+After the cluster installation is complete and the cluster becomes `Ready`, a `ClusterGroupUpgrade` CR corresponding to this cluster, with a list of ordered policies defined by the `ran.openshift.io/ztp-deploy-wave annotations`, is automatically created by the {cgu-operator}. The cluster's policies are applied in the order listed in `ClusterGroupUpgrade` CR.
++
+You can monitor the high-level progress of configuration policy reconciliation using the following commands:
++
+[source,terminal]
+----
+$ export CLUSTER=<clusterName>
+----
++
+[source,terminal]
+----
+$ oc get clustergroupupgrades -n ztp-install $CLUSTER -o jsonpath='{.status.conditions[?(@.type=="Ready")]}'
+----
+
+. You can monitor the detailed cluster policy compliance status using the {rh-rhacm} dashboard or the command line.
+
+.. To check policy compliance using the command line, run the following command:
++
+[source,terminal]
+----
+$ oc get policies -n $CLUSTER
+----
++
+.Example output
+[source,terminal]
+----
```

Comment:
The output should have included some enforce policies that are created by CGU since this step is to monitor the progress. And should have appropriate compliance states.
Or perhaps just remove the output from doc.

Reply:
How does this look:
```
NAME                                                             REMEDIATION ACTION   COMPLIANCE STATE   AGE
ztp-cnfde10-policies.cnfde10-config-policy                       inform               Compliant          19h
ztp-common-cnfde10.common-cnfde10-amq-sub-policy                 inform               Compliant          19h
ztp-common-cnfde10.common-cnfde10-bare-metal-events-sub-policy   inform               Compliant          19h
ztp-common-cnfde10.common-cnfde10-config-amq-policy              inform               Compliant          19h
ztp-common-cnfde10.common-cnfde10-config-events-policy           inform               Compliant          19h
ztp-common-cnfde10.common-cnfde10-config-policy                  inform               Compliant          19h
ztp-common-cnfde10.common-cnfde10-subscriptions-policy           inform               Compliant          19h
ztp-group-cnfde10.group-cnfde10-config-policy                    inform               Compliant          19h
ztp-group-cnfde10.group-cnfde10-perf-policy                      inform               Compliant          19h
```